core/src/plugins/filed/gfapi/gfapi-fd.cc

```diff
@@ -1738,11 +1738,6 @@ static bRC pluginIO(PluginContext* ctx, struct io_pkt* io)
 
   switch (io->func) {
     case IO_OPEN:
-      /*
-       * Close the gfd when it was not closed before.
-       */
-      if (p_ctx->gfd) { glfs_close(p_ctx->gfd); }
```

Comment:
Is this codepath really not needed anymore?
As far as I understand, p_ctx->gfd would be NULL in the case of the error that is fixed, 
so this glfs_close() call would be skipped anyway.

Reply:
Yes, as p_ctx->gfd is set to NULL unconditonally now on IO_CLOSE, the mentioned codepath would be never called.